docs/cog_guides/cog_manager_ui.rst

```diff
@@ -171,16 +171,16 @@ removepath
 
 .. code-block:: none
 
-    [p]removepath <path_number>
+    [p]removepath <path_number...>
 
 **Description**
 
-Removes a path from the list of available paths. Its cogs won't be accessible
-anymore.
+Removes one or more path from the list of available paths. Its cogs won't be
+accessible anymore.
 
 **Arguments**
 
-*   ``<path_number>``: The number of the path to remove. You can get it with
+*   ``<path_number>``: The number of the path(s) to remove. You can get it with
     the :ref:`paths <cogmanagerui>` command.
```

Comment:
```suggestion
    the :ref:`paths <cogmanagerui-command-paths>` command.
```
I know you didn't add this reference, but I think we could make the reference more accurate by being direct to `[p]paths` label instead of the base of the document.

If you could also apply this same change to the documentation for `[p]removepaths`? I can't make a suggestion for you on that seeing as it's not on the diff.